src/app/lib/utilities/audioVideo/mediaPlayerSettings.js

```diff
@@ -0,0 +1,80 @@
+import mediatorURL from './mediatorUrl';
+import filterForBlockType from '../blockHandlers';
+import deepGet from '../deepGet';
+
+const mediaPlayerSettings = ({
+  aresMediaBlocks,
+  env,
+  statsDestination,
+  statsPageIdentifier,
+}) => {
+  if (!aresMediaBlocks) {
+    return null;
+  }
+
+  const aresMediaMetadata = filterForBlockType(
+    aresMediaBlocks,
+    'aresMediaMetadata',
+  );
+  const pid = deepGet(['model', 'id'], aresMediaMetadata);
+  const nestedModel = deepGet(['model'], aresMediaMetadata);
+  const kind =
```

Comment:
Should we extract these methods into a separate file, I see some are being used here too:

https://github.com/bbc/simorgh/blob/a9c974e68f1c0a05a7988416623ce59e32dcc305/src/app/containers/AudioVideo/index.jsx#L57-L64

Reply:
Will be looked at in https://github.com/bbc/simorgh/issues/2412